medical-practitioner-portal/src/app/get-help/get-help.component.html

```diff
@@ -13,7 +13,7 @@ <h1>Getting help with Driver Medical Fitness</h1>
     </p>
   </div>
   <div class="help-cards">
-    <div>
+    <div style="width: 385px">
```

Comment:
curious, do we have CSS standards? I don't have a strong opinion about inline CSS, I think it makes sense if used appropriately, especially for small changes like this